src/Components/Ennumerated_Components/Book.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+function Book(props) {
+  const { book } = props;
+  const { title, category, progress, author } = book;
+
+  return (
+    <div className="notshown Book">
+      <div className="col">
+        <span className="fw-700">{category}</span>
+        <span>{title}</span>
+        <span>{author}</span>
+        <span className="sidemenu">
+          <span>Remove</span>
+          <span>Edit</span>
+        </span>
+      </div>
+      <div className="col">
+        <span>PROGRESS</span>
+        <span>
+          Update Progress&nbsp;
+          {progress}
+          %
+        </span>
+      </div>
+    </div>
+  );
+}
+
+const mapDispatchToProps = dispatch => ({
+  // dispatching plain actions
+  // CREATE_BOOK: (book) => dispatch({ type: 'CREATE_BOOK', book }),
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))